packages/react/src/components/UIShell/HeaderMenu.js

```diff
@@ -221,6 +221,9 @@ class HeaderMenu extends React.Component {
   };
 }
 
-export default React.forwardRef((props, ref) => {
+const Component = React.forwardRef((props, ref) => {
```

Comment:
While verbose, would something like this make? Just thinking out loud. `Component` can conflict with `React.Component` if folks pull it off the import so would hate to run into that.

```suggestion
const HeaderMenuForwardRef = React.forwardRef((props, ref) => {
```